src/main/webapp/timestamps.js

```diff
@@ -31,9 +31,16 @@ function timestampToString(timestamp) {
 }
 
 /**
- * Returns the total number of seconds since the start of a video at a
- * certain {@code timestampMilliseconds}.
+ * Converts a number of {@code seconds} into a integer number of
+ * milliseconds.
  */
-function timestampToSeconds(timestampMilliseconds) {
-  return timestampMilliseconds / 1000;
+function secondsToMilliseconds(seconds) {
```

Comment:
Can you rename seconds to timestamp or something more descriptive?

From the function name we know it is seconds

Reply:
This was meant to be a generic time converting util function.  Once we fix #158 it should be removed.  I'll add a TODO